docs/contributors/getting-started-native-mobile.md

```diff
@@ -0,0 +1,131 @@
+# Getting Started for the React Native based Mobile Gutenberg
+
+This is the mobile version of [Gutenberg](https://github.com/WordPress/gutenberg), targeting Android and iOS. It's a React Native library bootstrapped by [Create React Native App](https://github.com/expo/create-react-native-app) and now ejected.
```

Comment:
> This is

Not sure the "this is" fits well in here. I think it is at the top of [the gutenberg-mobile repo](https://github.com/wordpress-mobile/gutenberg-mobile) but not here. WDYT about the following suggestion? :

> Welcome! This it the Getting Started guide for the native mobile port of the block editor, targeting Android and iOS devices. Overall, it's a React Native library to be used in parent greenfield or brownfield apps. Continue reading for information on how to build and it!

Reply:
I like this! Added here: https://github.com/WordPress/gutenberg/pull/23951/commits/92e62d3a35516cda0958547f5b496df3b483b7ed

Reply:
Well, turns out the suggestion had a couple of issues on its own 😬. Can you follow up with revising @cameronvoell , even though the PR got merged? Here are the issues I notice:

* "Overall, it's a React..." -> "Overall, the block editor is developed as a React...", otherwise the "it's" seems to refer to the "guide", which is obviously not a RN library.
* "how to build and it!" -> "how to build and run it".

Thanks!